toptal/picasso
# feat: [FX-1387] Add Switch component

[FX-1387](https://toptal-core.atlassian.net/browse/FX-1387)

### Description

Adds the Switch component to picasso-lab based on MUI Switch

### How to test

- Run storybook and navigate to Lab Switch
- Make sure the component styles match the design
- Make sure the behavior acts as intended
- Focus the switch by pressing tab and test style / behavior by pressing the spacebar 

### Screenshots

![image](https://user-images.githubusercontent.com/254110/98975240-883edd80-251e-11eb-8e3a-024b353c574b.png)


### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run  whole pipeline
- `@toptal-bot run danger` - Danger checks
- `@toptal-bot run lint` - Run linter
- `@toptal-bot run test` - Run jest
- `@toptal-bot run build` - Check build
- `@toptal-bot run test:visual` or `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>
